samples/excel/90-scenarios/currency-converter.yaml

```diff
@@ -0,0 +1,178 @@
+order: 5
+id: excel-scenarios-currency-converter
+name: Currency Converter
+description: 'Reads rows and update base currency column, based on value date.'
```

Comment:
Let's change the description to better highlight the external API being used.
"Uses an exchange rate API to convert currency values based on their original transaction times."